hip/components/memory.hip.hpp

```diff
@@ -144,6 +144,13 @@ __device__ __forceinline__ ValueType load_relaxed_shared(const ValueType* ptr)
 }
 
 
+template <typename ValueType>
+__device__ __forceinline__ ValueType load_relaxed_local(const ValueType* ptr)
+{
+    return load_generic<__ATOMIC_RELAXED, HIP_SCOPE_THREADBLOCK>(ptr);
```

Comment:
This is the same as for `load_relaxed_shared`. I guess HIP doesn't have a finer granularity to distinguish those.

Reply:
they might have a different granularity on the code generation side, since I assume there are dedicated instructions for share memory access, and pointers are likely annotated with `__shared__` internally.